mire_detection.py

```diff
@@ -355,16 +364,16 @@ def cleanup_plots_heuristics(r_pixels, start_angle, end_angle, jump, n_mires, ou
         r_pixels_cleaned.append(r_pixels_angle)
 
     # Reshaping from angle x mires to mires x angle 
-    r_pixels = []
-    for mire in range(n_mires):
-        r_pixels_temp = []
-        for idx, angle in enumerate(np.arange(start_angle, end_angle, jump)):
-            r_pixels_temp.append(r_pixels_cleaned[idx][mire])
-        plt.plot(np.arange(start_angle, end_angle, jump), r_pixels_temp, ls='-', label=str(mire))
-        r_pixels.append(r_pixels_temp)
-    plt.legend()
-    plt.savefig(output_folder+'/'+image_name+'/plots-modified.png')
-    plt.close()
+    # r_pixels = []

```

Comment:
What is this change? Seems unrelated to the dl based mire detection?

Reply:
just removed some old plots that are not relevant